templates/apps/sumo-aws-apps/common-org/common-org-management.yaml

```diff
@@ -952,6 +868,158 @@ Resources:
       SumoAccessKey: !Ref SumoAccessKey
       SumoDeployment: !Ref SumoDeployment
 
+  OrganizationsRule:
+    Condition: create_config_recorder_role  
+    Type: AWS::Events::Rule
+    Properties:
+      Name: !Sub ${ParentStackName}-org-update
+      Description: Config Trigger on Organizations update
+      EventPattern:
+        source:
+          - aws.organizations
+        detail-type:
+          - AWS API Call via CloudTrail
+        detail:
+          eventSource:
+            - organizations.amazonaws.com
+          eventName:
+            - AcceptHandshake
+            - CreateAccountResult
+            - RemoveAccountFromOrganization
+      State: ENABLED
+      Targets:
+        - Arn: !GetAtt rSumoLogicLambdaHelper.Arn
+          Id: !Sub CommonOrgManagement-SumoLogicLambdaHelper-${ParentStackName}
+
+  PermissionForOrganizationsRuleToInvokeLambda:
+    Condition: create_config_recorder_role  
+    Type: AWS::Lambda::Permission
+    Properties:
+      FunctionName: !GetAtt rSumoLogicLambdaHelper.Arn
+      Action: lambda:InvokeFunction
+      Principal: events.amazonaws.com
+      SourceArn: !GetAtt OrganizationsRule.Arn
+
+  CrossRegionEventRuleRole:
+    Type: AWS::IAM::Role
+    Condition: NotGlobalRegionUsEast1
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W28
+            reason: Specific role name provided
+    Properties:
+      RoleName: !Sub "config-recorder-start-${ParentStackName}"
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Action: sts:AssumeRole
+            Principal:
+              Service:
+                - events.amazonaws.com
+      Policies:
+        - PolicyName: config-events
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: events:PutEvents
+                Resource: !Sub arn:${AWS::Partition}:events:${AWS::Region}:${AWS::AccountId}:event-bus/default
```

Comment:
instead of ${AWS::Region} shouldn't it be all the config regions (where config is enabled) ? Did we test with a region other that us-east-1 whether the default event-bus is getting the events cross region?

Reply:
fixed in last commit. i tested with a region other that us-east-1, the default event-bus is getting the events cross region